### PR TITLE
Adds the Laravel set lists

### PR DESCRIPTION
**Pull request by peterfox**
Added to the Laravel set list for #2556

### CODE REVIEW

File: src/RuleFilter/Bridge/LaravelSetProvider.php [hunk 1]
```diff
@@ -8,13 +8,41 @@
...
 /**
  * @todo should be part of the Laravel package, move later
  * @see https://github.com/driftingly/rector-laravel/tree/main/config/sets
  */
 final class LaravelSetProvider implements SetProviderInterface
 {
+    /**
+     * @var string[]
+     */
+    private static array $laravelFiveVersions = [
```
Comment (samsonasik) [inline src/RuleFilter/Bridge/LaravelSetProvider.php:22]: use constant is possible?

**Comment by TomasVotruba** [comment]
I'll look into this next week, thank you Peter!

**Comment by TomasVotruba** [comment]
Hey, can you re-open PR, and target current `main`?

Thanks :+1:

**Comment by TomasVotruba** [comment]
So far so good :) 

https://getrector.com/find-rule?activeRectorSetGroup=laravel&rectorSet=laravel-code-quality

**Comment by peterfox** [comment]
@TomasVotruba opened up #2579

**Pull request closed without merging**